wiki/Community/Mappers_Guild/en.md

```diff
@@ -38,9 +38,11 @@ Priority quests are labelled by difficulty. Tier 1 is the easiest and most commo
 
 ### Rewards
 
-![Mappers' Guild profile badges](img/mg-badges.png "The four tiers of the Mappers' Guild profile badge")
+![Mappers' Guild profile badges](img/mg-badges.png "The five tiers of the Mappers' Guild profile badge")
```

Comment:
to prevent cache issues

```suggestion
![Mappers' Guild profile badges](img/mg-badges.png?2024-02 "The five tiers of the Mappers' Guild profile badge")
```